projects/github-actions/pr-is-up-to-date/changelog/update-monorepo-master-refs-primary

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+Renaming master references to trunk in GH action files, to work with the GitHub default branch name changing to trunk.
```

Comment:
Thanks for explaining this! Since we're going through everything, I'll change it all to main (except in those two repo-gardening files). I'll commit your suggestion here too.